gpsutils/serial_data_reader.go

```diff
@@ -80,7 +81,10 @@ func (dr *SerialDataReader) backgroundWorker(cancelCtx context.Context) {
 
 		line, err := r.ReadString('\n')
 		if err != nil {
-			dr.logger.CErrorf(cancelCtx, "can't read gps serial %s", err)
+			if readErrCount < 5 {
+				dr.logger.CDebugf(cancelCtx, "can't read gps serial %s", err)
```

Comment:
from ticket I assumed this would be error at first, then debug after 5

Reply:
added!